src/Commands.js

```diff
@@ -37,6 +38,7 @@
             fired = [],
             prefix = "cmdId_",
             indexer = 0,
+            rethrow, //default true
```

Comment:
Since it is an event platform it needs to protect all the consumers of each event.
As such, it can't throw exceptions in any case since it will break other flows.
It is on you to wrap your "risky'" code bits with your own try - catch mechanism. 
This is our philosophic take in this specific implementation due to the way we use it across all our production services.